pymc/stats/log_likelihood.py

```diff
@@ -120,6 +120,7 @@ def compute_log_likelihood(
         coords={
             cname: np.array(cvals) if isinstance(cvals, tuple) else cvals
             for cname, cvals in model.coords.items()
+            if cvals is not None
```

Comment:
We should have a utility that does this dims/coords conversion, because this function shouldn't have to know these gotchas. This doesn't need to be done in this PR, but we should open an issue for it.